src/components/common/ChartCard.tsx

```
@@ -28,6 +30,11 @@ const ChartCard = ({ title, headers, imageSrc, message }: ChartCardProps) => {
     setWeeklyFilter,
   } = useContext(Context);
 
+  const { data, isLoading, isError } = useQuery({
```

ESLint

[error] 'data' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)